apps/nextjs/src/components/AppComponents/Chat/chat-left-hand-side.tsx

```diff
@@ -15,6 +17,7 @@ type ChatLeftHandSideProps = {
   isDemoLocked: boolean;
   showLessonMobile: boolean;
   setShowLessonMobile: (value: boolean) => void;
+  demo: DemoContextProps;
```

Comment:
What's the demo thing being used for here?

Reply:
There is an inchat button that needs blocking